app/services/PenaltiesService.scala

```diff
@@ -36,9 +36,11 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) {
   }
 
   def isAnyLSPUnpaidAndSubmissionIsDue(penaltyPoints: Seq[PenaltyPoint]): Boolean = {
-    penaltyPoints.exists(penalty => penalty.status == PointStatusEnum.Due &&
-      penalty.`type` == PenaltyTypeEnum.Financial &&
-      penalty.period.isDefined && penalty.period.get.submission.submittedDate.isEmpty && !penalty.appealStatus.contains(AppealStatusEnum.Accepted) && !penalty.appealStatus.contains(AppealStatusEnum.Accepted_By_Tribunal))
+    penaltyPoints.exists(penalty => penalty.status == PointStatusEnum.Due
+      && penalty.`type` == PenaltyTypeEnum.Financial && penalty.period.isDefined
+      && penalty.period.map(penaltyPeriod => penaltyPeriodHelper.sortedPenaltyPeriod(penaltyPeriod).head.submission.submittedDate.isEmpty).get
```

Comment:
In the above suggested code, what is 'identity' used in fold

Reply:
`identity` takes one argument (which can be omitted) and returns the same argument. It saves time and is useful in functions like these to prevent having things like `x => x`.

So it just replaces the need of doing `x => x`